packages/experimental/Icon/src/Icon.tsx

```diff
@@ -54,10 +54,11 @@ function renderSvg(iconProps: IconProps) {
   const { width, height } = iconProps;
   const viewBox = iconProps.svgSource.viewBox;
 
+  // react-native-svg is still on 0.61, and their color prop doesn't handle ColorValue
```

Comment:
This basically casts to string right now, so it might still be a blocker. Another option is to only allow strings for icon color tokens, but it might be better to just wait for this to come along for now, since I have other blockers.

Reply:
Revisiting this: I think a cast will crash and we should separate out the icon tokens and allow them to only be a number or string. As it is implemented rn, I think the icon will inherit the text color token if no icon color is supplied

Reply:
Would it be an issue if I added a color prop to SvgIconProps which was string only but left the color prop alone so that font icons could take advantage of the ColorValue change? I guess it could break things downstream so I guess that's not the best solution.

Reply:
Addressed with latest commit